src/components/client/campaigns/CampaignDetails.tsx

```diff
@@ -111,6 +133,15 @@ export default function CampaignDetails({ campaign }: Props) {
         campaign={campaign}
         showExpensesLink={(expensesList && expensesList?.length > 0) || canEditCampaign}
       />
+      {subscribeIsOpen && <RenderSubscribeModal setOpen={setSubscribeOpen} campaign={campaign} />}
+      <Grid item textAlign="center" pr={15}>
+        <Button
```

Comment:
The subscribe button starts breaking at viewports<400px

Reply:
The subscribe button looks too big there at the beginning, I was even thinking if it could be styled as a Link Button or even a Checkbox in the side card with the Donations Progress, because that is where the most updates happen anyway.